stable/grafana/templates/deployment.yaml

```diff
@@ -13,6 +13,8 @@ spec:
   template:
     metadata:
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
[None of the other official charts](https://github.com/kubernetes/charts/search?p=1&q=checksum%2Fconfig&type=&utf8=✓) that use the same `checksum/config` annotation hack has any option in their `.Values` file in order to configure this.